workload/s3/s3api/multipart_partcopy_random.yaml

```diff
@@ -139,7 +139,7 @@ test_11:
     min_runtime: 5d
     sessions: 30
     tool: s3api
-    operation: multipart_partcopy
+    operation: multipart_partcopy_random
```

Comment:
Make sure function mapping updated for this changes.

Reply:
Done